logdna/resource_stream_config.go

```diff
@@ -0,0 +1,175 @@
+package logdna
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+const streamConfigID = "stream"
```

Comment:
this is something I'm not 100% sure about. this is essentially a singleton resource, it's only possible to configure one set of settings. there is no ID so this string is used instead.

the only thing this impacts is `terraform import`, where you need to specify `stream` as the ID you are importing.

Reply:
It seems like this is the only way to do this without have a cumbersome wrapper around `terraform import`, which is bad.  Users need to be able to use the native terraform commands.

Reply:
yeah the `terraform import` flow is definitely only meant for targeting single resources. after getting it working here, and playing with it a lot, I've found it to be different than I was expecting (as a casual, non-exprt terraform user at least).

this page clarifies the expected workflow a bit
https://learn.hashicorp.com/tutorials/terraform/state-im